### PR TITLE
Add AssertionInterface

### PR DESCRIPTION
**Pull request by pathmissing**

### CODE REVIEW

File: src/AppserverIo/Psr/MetaobjectProtocol/Dbc/Assertions/AssertionInterface.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * \AppserverIo\Doppelgaenger\Interfaces\AssertionInterface
```
Comment (wick-ed) [inline src/AppserverIo/Psr/MetaobjectProtocol/Dbc/Assertions/AssertionInterface.php]: wrong namespace

File: src/AppserverIo/Psr/MetaobjectProtocol/Dbc/Assertions/AssertionInterface.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * PHP version 5
+ *
+ * @author    Bernhard Wick <bw@appserver.io>
+ * @copyright 2015 TechDivision GmbH - <info@appserver.io>
```
Comment (wick-ed) [inline src/AppserverIo/Psr/MetaobjectProtocol/Dbc/Assertions/AssertionInterface.php]: please add yourself as author and update the copyright time

File: .travis.yml [hunk 3]
```diff
@@ -1,13 +1,17 @@
 language: php
 
 php:
+  - 7.0
```
Comment (wick-ed) [inline .travis.yml:4]: was the travis.yml change discussed with core contributors?

Comment (pathmissing) [reply .travis.yml:4]: These changes were approved by one of the core contributors

Comment (wick-ed) [reply .travis.yml:4]: ok then :)

**Pull request merged**